src/ml/neural_net/mps_layer_conv_padding.h

```diff
@@ -0,0 +1,36 @@
+#ifndef mps_layer_conv_padding_h
+#define mps_layer_conv_padding_h
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+
+API_AVAILABLE(macos(10.14))
+@interface ConvolutionPadding : NSObject <MPSNNPadding>
+{
+  int mPaddingWidth;
+  int mPaddingHeight;
+  int mStrideWidth;
+  int mStrideHeight;
+}
+
+@property (class, readonly) BOOL supportsSecureCoding;
+
+- (id) initWithParams:(int)paddingWidth
```

Comment:
Initializers in modern Objective C code should return `instancetype`. Also, let's use idiomatic Objective C integer types, like NSUInteger, or whatever the underlying MPS API expects.

Reply:
done!